modules/fakemeta/pdata_gc.cpp

```diff
@@ -75,20 +75,40 @@ static BaseFieldType GetBaseDataType(TypeDescription &data)
 	return BaseFieldType::None;
 }
 
-#define GET_TYPE_DESCRIPTION(position, data, baseType)                                     \
+
+#define GET_TYPE_DESCRIPTION(position, data, baseType, list)  \
```

Comment:
I love me some macro trickery but I think with these changes we can remove this now?